src/bindings/python/src/pyopenvino/core/common.cpp

```diff
@@ -76,16 +106,121 @@ const TensorIndexMap cast_to_tensor_index_map(const py::dict& inputs) {
 }
 };  // namespace containers
 
+namespace string_helpers {
+
+py::array bytes_array_from_tensor(ov::Tensor&& t) {
+    if (t.get_element_type() != ov::element::string) {
+        OPENVINO_THROW("Tensor's type must be a string!");
+    }
+    // Slow approach ~x5:
+    // py::list _list;
+    // for(size_t i = 0; i < t.get_size(); ++i) {
+    //     PyObject* _unicode_obj = PyBytes_FromStringAndSize(&data[i][0], data[i].length());
+    //     _list.append(_unicode_obj);
+    //     Py_XDECREF(_unicode_obj);
+    // }
+    // return py::array(_list);
```

Comment:
Should we keep this in the codebase? From what I understand a faster method is implemented, is there a reason to have the slower approach around?

Reply:
Leftover to be removed.

Reply:
Done.